sycl/source/detail/xpti_registry.hpp

```diff
@@ -27,11 +27,6 @@ namespace detail {
 // We define a sycl stream name and this will be used by the instrumentation
 // framework
 inline constexpr const char *SYCL_STREAM_NAME = "sycl";
-// Stream name being used for traces generated from the SYCL plugin layer
-inline constexpr const char *SYCL_PICALL_STREAM_NAME = "sycl.pi";
-// Stream name being used for traces generated from UR calls. This stream
-// contains information about function arguments.
-inline constexpr const char *SYCL_PIDEBUGCALL_STREAM_NAME = "sycl.pi.debug";
```

Comment:
These were to be renamed but I didn't see any usage of them throughout the project. Can someone let me know if they are used in some other way and should be kept and renamed.

Reply:
I think it was removed during pi plugin removal. These were streams for pi calls notifications.

Reply:
Thanks good to know they aren't used.